pyeudiw/storage/mongo_storage.py

```diff
@@ -270,6 +271,18 @@ def add_trust_attestation(self, entity_id: str, attestation: list[str], exp: dat
             "trust_attestations", entity_id, updated_entity, exp
         )
 
+    def add_trust_attestation_metadata(self, entity_id: str, metadata: dict):
+        entity = self._get_trust_attestation("trust_attestations", entity_id)
+
+        if entity is None:
+            raise ValueError(
+                f'Document with entity_id {entity_id} not found.'
+            )
+        
+        entity["metadata"] = metadata
```

Comment:
as said yesterday

entity['metadata'][$metadata_type] :dict

where $metadata_type is set of: openid_credential_issuer, wallet_provider, wallet_relying_party and so on ...

Reply:
Resolved in [1d3fa1b](https://github.com/italia/eudi-wallet-it-python/pull/188/commits/1d3fa1b74b7251c4f4702dabb7aedcc984824572)